docs/download_links.md

```diff
@@ -0,0 +1,60 @@
+# Download Links
+
+This repo also serves the vital purpose of handling download redirects.
+
+Currently we build on every push to `master` for `pulsar`. Each new run on Cirrus CI results in a binary application for many different operating systems and their variances.
+
+In order to allow users to install these easily the `https://web.pulsar-edit.dev/download` URL can redirect to any one of our binaries through a simple link and simple query parameters.
+
+To configure a link to properly point a user in the right direction take a look at the content below:
+
+---
+
+URL: https://web.pulsar-edit.dev/download
+Query Parameters: `os` && `type`
+
+All `os` Options:
+
+- `windows`
+- `intel_mac`
+- `silicon_mac`
+- `linux`
+- `arm_linux`
+
+All `type` Options:
+
+- `windows_setup`
+- `windows_portable`
+- `windows_blockmap`
+- `mac_zip`
+- `mac_zip_blockmap`
+- `mac_dmg`
+- `mac_dmg_blockmap`
+- `linux_appimage`
+- `linux_tar`
+- `linux_rpm`
+- `linux_deb`
+
+Full Table:
```

Comment:
```suggestion
Example: https://web.pulsar-edit.dev/download?os=windows&type=windows_portable

Full Table:
```
I know it is kind of obvious but if it saves somebody 30 seconds of googling how to make a url query then I think it is worth it.

Reply:
Good idea on that one, like we always preach, lowest barrier to entry at all times